x-pack/test/functional/services/ml/job_wizard_common.ts

```diff
@@ -335,6 +335,37 @@ export function MachineLearningJobWizardCommonProvider(
       }
     },
 
+    async assertStartDatafeedSwitchExists() {
+      const subj = 'mlJobWizardStartDatafeedCheckbox';
+      await testSubjects.existOrFail(subj, { allowHidden: true });
+    },
+
+    async getStartDatafeedSwitchCheckedState(): Promise<boolean> {
+      const subj = 'mlJobWizardStartDatafeedCheckbox';
+      const isSelected = await testSubjects.getAttribute(subj, 'aria-checked');
```

Comment:
have you tried `testSubjects.isChecked(...)`?

Reply:
i can try it, this is a copy of the `getModelPlotSwitchCheckedState` function. 
@pheyos is `getAttribute(subj, 'aria-checked')` preferred over `isChecked` ?

Reply:
When I introduced the first service methods for switches, the `isChecked` didn't work for EUI switches and I had a discussion with the EUI folks about it. They advised to use the `aria-checked`. But that might as well have changed since then.

Reply:
i can confirm, `isChecked ` still doesn't work for switches.